src/gui/menu.cpp

```diff
@@ -594,10 +597,12 @@ Menu::draw(DrawingContext& context)
     const int text_width = static_cast<int>(Resources::normal_font->get_text_width(m_items[m_active_item]->get_help()));
     const int text_height = static_cast<int>(Resources::normal_font->get_text_height(m_items[m_active_item]->get_help()));
 
+    const float margin_y = 16.0f;
```

Comment:
This variable can be moved as a `static` constant on the top of the `.cpp` file.

`static const float HELP_MARGIN_Y = 16.f;`

Reply:
Thanks, I pushed a commit that changes this.